### PR TITLE
refactor(devices): rename devices view to clients

### PR DESCRIPTION
**Pull request by vladikoff**
@vbudhram @ryanfeeley r?

### CODE REVIEW

**Comment by vladikoff** [comment]
@ryanfeeley is `Devices & apps` good or should it be `Apps`?

**Comment by vladikoff** [comment]
#### Requires https://github.com/mozilla/fxa-content-experiments/pull/40

File: app/scripts/lib/router.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ define(function (require, exports, module) {
   var ConfirmView = require('../views/confirm');
   var CookiesDisabledView = require('../views/cookies_disabled');
   var DeleteAccountView = require('../views/settings/delete_account');
-  var DevicesView = require('../views/settings/devices');
+  var ClientsView = require('../views/settings/clients');
```
Comment (rfk) [inline app/scripts/lib/router.js]: 👍

File: app/scripts/views/settings/clients.js [hunk 2]
```diff
@@ -105,6 +107,18 @@ define(function (require, exports, module) {
       });
     },
 
+    _getPanelTitle: function () {
+      var title = t('Devices');
+
+      if (this._able.choose('appsListVisible', {
```
Comment (vbudhram) [inline app/scripts/views/settings/clients.js]: Should you check for the query existence and if it is `true` before displaying?

Comment (vladikoff) [inline app/scripts/views/settings/clients.js]: The code here should do it: https://github.com/mozilla/fxa-content-experiments/pull/40/files#diff-ef4b3a9aa6190a2bd91842a4370aabddR9

**Comment by vbudhram** [comment]
@vladikoff Should https://github.com/mozilla/fxa-content-server/pull/4055/files#diff-e8681b432d4785a0fcec38546b6b400eR18 and https://github.com/mozilla/fxa-content-server/pull/4055/files#diff-e8681b432d4785a0fcec38546b6b400eL32 say `Devices and apps`?

Otherwise, LGTM 👍

**Comment by vladikoff** [comment]
> @vladikoff Should https://github.com/mozilla/fxa-content-server/pull/4055/files#diff-e8681b432d4785a0fcec38546b6b400eR18 and https://github.com/mozilla/fxa-content-server/pull/4055/files#diff-e8681b432d4785a0fcec38546b6b400eL32 say Devices and apps?

Yeap, the first one should 👍

**Comment by ryanfeeley** [comment]
@vladikoff 
* Devices if devices feature not ready
* Apps if only apps has launched
* Devices & apps if both have launched

**Comment by rfk** [comment]
> Apps if only apps has launched

ISTM this is not a situation we will get ourselves into in practice?

**Pull request merged**